docs/dev/pdf-format.md

```diff
@@ -1,7 +1,16 @@
 # The PDF Format
 
 It is recommended to look in the PDF specification for details and clarifications.
-This is only intended to give a very rough overview of the format.
+
+[PDF Specification Archive](https://pdfa.org/resource/pdf-specification-archive/)
```

Comment:
These entries probably should take advantage of an unordered list - otherwise, this looks a bit odd on the rendered page: https://pypdf--2983.org.readthedocs.build/en/2983/dev/pdf-format.html